### ISSUE TITLE
Huge power consumption since last updates 3.6.19 / 3.6.20

### ISSUE THREAD

**Issue by moisesbelda** [ValveSoftware/SteamOS#1724]
#### Your system information

* Steam client version: 1726693985
* SteamOS version: 3.6.20
* Opted into Steam client beta?: [No] 
* Opted into SteamOS beta?: [No] 
* Have you checked for updates in Settings > System?: [Yes]

#### Please describe your issue in as much detail as possible:
Since the last update of Steam OS (3.6.20, Im on the stable channel) I've been noticing a significant increase in battery consumption on the Steam Deck. I've tested it with apps that have minimal battery usage, like remote streaming apps (such as XBPlay or Moonlight), and even just in the main menu, without running any streams. The stats show it's consuming nearly 10W. This is outrageous considering nothing is being processed, just the app's menu. Before this update, I used to play remote streaming games with these apps, and the consumption didn't even reach 5W. Also, if the steam deck is idle, I notice that the battery is being consumed more than ever. Of course rest of games have increased the power consumption, but these cases with remote play apps are the most outrageous cases.

I've rollbacked to 3.5.19, and the issue is still there, so as per what is commented in #1689, issue seems to be related to the firmware F7A0131, but I don't know how to rollback firmware

#### Steps for reproducing this issue:

1.  Open any app with minimal battery usage (XBPlay or Moonlight) 
2.  Check the power usage in the menu of the app
3. 8% - 10% without using the app. Inside a stream, 10%-12%.

**Comment by positiveoctopus** [comment]
> #### Your system information
> * Steam client version: 1726693985
> * SteamOS version: 3.6.20
> * Opted into Steam client beta?: [No]
> * Opted into SteamOS beta?: [No]
> * Have you checked for updates in Settings > System?: [Yes]
> 
> #### Please describe your issue in as much detail as possible:
> Since the last update of Steam OS (3.6.20, Im on the stable channel) I've been noticing a significant increase in battery consumption on the Steam Deck. I've tested it with apps that have minimal battery usage, like remote streaming apps (such as XBPlay or Moonlight), and even just in the main menu, without running any streams. The stats show it's consuming nearly 10W. This is outrageous considering nothing is being processed, just the app's menu. Before this update, I used to play remote streaming games with these apps, and the consumption didn't even reach 5W. Also, if the steam deck is idle, I notice that the battery is being consumed more than ever. Of course rest of games have increased the power consumption, but these cases with remote play apps are the most outrageous cases.
> 
> I've rollbacked to 3.5.19, and the issue is still there, so as per what is commented in #1689, issue seems to be related to the firmware F7A0131, but I don't know how to rollback firmware
> 
> #### Steps for reproducing this issue:
> 1. Open any app with minimal battery usage (XBPlay or Moonlight)
> 2. Check the power usage in the menu of the app
> 3. 8% - 10% without using the app. Inside a stream, 10%-12%.

same, do you have an OLED or LCD model?

**Comment by moisesbelda** [comment]
> > Replying to [#1724 (comment)](https://github.com/ValveSoftware/SteamOS/issues/1724#issue-2674687442)
> 
> same, do you have an OLED or LCD model?

LCD

**Comment by positiveoctopus** [comment]
yeah it seems like the LCD model is the most affected from these bugs

Il giorno ven 29 nov 2024 alle ore 15:38 Moisés ***@***.***>
ha scritto:

**Comment by AlchemicSoul** [comment]
im on stock settings on an oled model and my performance has sunk in so many ways.

menu buttons can display ghosted frames and lag for a couple seconds. power off doesn't actually power off every time. lag on volume buttons occasionally. lag on most menus. lower battery life in most games. etc. 

went from silk to feeling like a switch with a dying battery at times.

**Comment by nightspd** [comment]
> > Replying to [#1724 (comment)](https://github.com/ValveSoftware/SteamOS/issues/1724#issue-2674687442)
> 
> same, do you have an OLED or LCD model?

LCD

**Comment by xkarbin** [comment]
LCD user here, same issue.
